packages/docs/docs/cli/external-fhir-servers.md

```diff
@@ -19,34 +19,87 @@ To enable these scenarios, you will need a `clientId` or `clientSecret` to acces
 
 The examples below use the [CLI optional flags](/docs/cli#optional-flags).
 
+## Setting up your credentials
+
+Medplum CLI stores credentials to be used in a future period without needing it to be entered in every command. By using the `profile` command, this helps with the ability to work with multiple FHIR servers.
+
+### Example: Setting a Profile
+
+In this example, we will set up a profile using `medplum profile set <profileName>` with the flags below:
+
+- --auth-type
+- --client-id
+- --client-secret
+- --base-url
+- --token-url
+- --authorize-url
+- --fhir-url-path
+
+```bash
+medplum profile set <profileName> --base-url <base-url> --fhir-url-path <fhir-url-path> --token-url <token-url> --client-id <client-id> --client-secret <client-secret>
```

Comment:
Let's call this "Syntax"

Can we use backslashes here to make this a multiline string?

```bash
medplum profile set <profileName> \
    --base-url <base-url> \
    --fhir-url-path <fhir-url-path> \
    --token-url <token-url> \
    --client-id <client-id> \
    --client-secret <client-secret>
```

And then we can have another block labeled "Example" with dummy values.

I think @reshmakh  is correct, that we will want clearly labeled examples for each of the main auth methods.

----

Example: Basic Auth

```bash
medplum profile set "example" \
    --base-url "https://api.example.com" \
    --fhir-url-path "fhir/R4" \
    --auth-type "basic" \
    --client-id "MY_CLIENT_ID" \
    --client-secret "MY_CLIENT_SECRET"
```

Example: Client Credentials

```bash
medplum profile set "example" \
    --base-url "https://api.example.com" \
    --fhir-url-path "fhir/R4" \
    --token-url "oauth2/token" \
    --auth-type "client_credentials" \
    --client-id "MY_CLIENT_ID" \
    --client-secret "MY_CLIENT_SECRET"
```

Example: User Login / Authorization Code

```bash
medplum profile set "example" \
    --base-url "https://api.example.com" \
    --fhir-url-path "fhir/R4" \
    --authorize-url "oauth2/authorize" \
    --token-url "oauth2/token"
```